SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -1,8 +1,8 @@
 %global debug_package %{nil}
 Summary:        NVIDIA container runtime hook
 Name:           nvidia-container-toolkit
-Version:        1.13.5
-Release:        7%{?dist}
+Version:        1.16.2
```

Comment:
I think the comment from #10636 to update `libnvidia-container.spec` in lockstep with this package applies here as well.

@henryli001, can you confirm?

Reply:
Yes this is correct.